tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
I think it's important for us to provide our own implementation of `babelHelpers`, and include `externalHelpers: true` in these configuration options (see [here](https://github.com/meteor/react-packages/blob/76d20f806db0f1f6cd41c7f968b3e5de5f814d4a/babel-compiler/babel.js#L27)). Otherwise Babel is going to inject a lot of redundant boilerplate code into every transpiled file.